common/persistence/sql/sqlplugin/sqlite/plugin.go

```diff
@@ -177,7 +189,14 @@ func buildDSNAttr(cfg *config.SQL) (url.Values, error) {
 				key, value,
 			)
 		}
-		parameters.Set(key, value)
+
+		if _, ok := queryParameters[key]; ok {
```

Comment:
```suggestion
		if _, isValidQueryParameter := queryParameters[key]; isValidQueryParameter {
```

Reply:
this is not idiomatic go, but I will defer to you.